include/boost/geometry/strategies/cartesian/buffer_join_round.hpp

```diff
@@ -83,15 +90,16 @@ private :
         {
             angle2 -= two_pi;
         }
+        PromotedType const dangle = angle1 - angle2;
```

Comment:
This name may be confusing. It's like if there was something dangling, like a spike below a polygon or a pointer.
I'd probably use `andle_diff`. Just saying, no need to change it.